stuff/doc/Português((Brasil)/tips.md

```diff
@@ -0,0 +1,42 @@
+# Bem-vindo ao Tahoma2D 1.5!
+
+A Versão 1.5 traz um grande número de novos recursos, aprimoramentos e ainda mais correções que estão listadas [aqui](https://github.com/tahoma2d/tahoma2d/releases/tag/v1.5).
+
+### Alterações na Timeline/Xsheet
+
+- Células e Colunas não possuem a barra de arrastar no canto por padrão.
+- Se preferir o antigo comportamento, habilite `Preferências` -> `Cena` -> `Mostrar Barras de Arrastar Colunas e Células`.
+- Clique em uma Célula exposta ou em uma Coluna não vazia para arrastar e movê-la.
+- Selecione uma Célula/Coluna vazia e arraste para iniciar uma seleção de múltiplas Células/Colunas.
+- `ALT` agora substitui `CTRL` para selecionar Quadros e Chaves de animação.
+- `CTRL` agora inicia uma seleção de múltiplas Células/Colunas quando o primeiro clique for em uma Célula/Coluna exposta.
+
+### Usuários de Windows
+
+Se estiver tendo problemas com canetas digitais nesta versão, especialmente se estiver usando um notebook-tablet/Tablet Surface tente uma ou as duas sugestões:
+- Habilite `Preferências` -> `Configurações de Toque/Tablet` -> `Usar suporte nativo do Qt para Windows Ink*` e reinicie o Tahoma2D.
+- Mantenha a opção acima ativa e instale os drivers `WinTab`. Cheque o fabricante do seu dispositivo para instalar o driver WinTab apropriado.
+
+-----
+
+## Atualizando do Tahoma2D 1.2 ou Migrando do OpenToonz
+
+Se estiver migrando do Tahoma2D 1.2 ou do OpenToonz, aqui estão algumas diferenças importantes de se mencionar:
+- Holds Implícitos
+  - Não há necessidade para extender o Quadro manualmente. Um Quadro exposto será mostrado até o próximo Quadro exposto.
+  - Use `Quadros de Parada` para terminar um hold sem expor outro Quadro.
+  - Clique com o botão direito do mouse sobre qualquer Célula e selecione `Habilitar/Desabilitar Holds Implícitos` para mudar os modos.
+  - Converta Cenas existentes de/para usar Holds Implícitos com `Cena` -> `Converter para Holds Implícitos/Explícitos`.
+- Algumas funções e configurações adicionais do OpenToonz estão escondidas.
+  - Habilite-as em `Preferências` -> `Geral` -> `Mostrar Preferências e Opções Avançadas*` e reinicie o Tahoma2D.
+
+-----
+## Dicas
+- Animações e recursos externos (desenhos) de uma Cena são salvos em arquivos separados dentro de uma pasta de Projeto.
+- Crie um Projeto `Tahoma2D` em sua pasta Documentos e use como seu projeto geral ao invés do padrão `sandbox`.
+- Use `Save All`, regularly, to save both scene and asset files. `Save Scene` does not save asset changes, just the scene changes.
```

Comment:
This line needs to be removed.

Reply:
Corrected! I was copying and then translating, by mistake left one of the originals in there xD